### PR TITLE
updating the test plan for study viewer tests

### PR DESCRIPTION
**Pull request by atharvar28**
Addition of more scenarios in the test plan like testing multiple datasets and more detailed steps while requesting access to study dataset

### CODE REVIEW

**Review by themarcelor — APPROVED** [review]
+1

**Review by themarcelor — APPROVED** [review]
+1

File: docs/testplans/study-viewer/study-viewer.md [hunk 1]
```diff
@@ -24,12 +24,21 @@ It enables to the user to check :
...
+  c. using `request_id` received, request to requestor service is send to update the status of the request to `APPROVED` <br>
+  d. user refreshes the study page and sees button disabled with message `DAR In Progress`
+  e. with the same `request_id`, another request to requestor service is send to update the status of the request to `SIGNED` <br>
+  f. user refreshes the study page and sees `Download` button displayed where the user can now download the study
+
+2. User with access - user logs in and goes to study page, click on `Download` button and download the file from indexd
  
 3. `Learn More` button navigates the user to `one row` page which displays the requested dataset
 
-4. Test functionality in a Data Commons with multiple configured Study Viewers
```
Comment (paulineribeyre) [inline docs/testplans/study-viewer/study-viewer.md:35]: you shouldn't remove `Test functionality in a Data Commons with multiple configured Study Viewers`. It was not implemented but it's still a test case that should be tested.

File: docs/testplans/study-viewer/study-viewer.md [hunk 2]
```diff
@@ -24,12 +24,21 @@ It enables to the user to check :
...
-a. go to page, click on `Request Access` button to request access, recives `request_id` from the Request Access Queue <br>
-b. makes a manual call to validate the `request_id` with the Requestor <br>
-c. go back to the back, click on `Download` button to download the dataset <br>
-
-2. User with access - go to page, click on `Download` button and download the file from indexd
+1.1. User does not login and requests the access <br>
+  a. go to study page, sees `Login to Request Access` button, clicks the button <br>
+  b. redirected to the `Login Page`. The user logs in with valid credentials  <br>
+  c. goes back to the study page, sees `Request Access` button to request access for the dataset <br>
+1.2. User logs in and request access <br>
```
Comment (paulineribeyre) [inline docs/testplans/study-viewer/study-viewer.md:31]: user already logged in during 1b

File: docs/testplans/study-viewer/study-viewer.md [hunk 3]
```diff
@@ -24,12 +24,21 @@ It enables to the user to check :
...
+  a. logs in and goes to the study page, clicks on `Request Access` button <br>
+  b. the user receives `request_id` <br>
+  c. using `request_id` received, request to requestor service is send to update the status of the request to `APPROVED` <br>
+  d. user refreshes the study page and sees button disabled with message `DAR In Progress`
+  e. with the same `request_id`, another request to requestor service is send to update the status of the request to `SIGNED` <br>
```
Comment (paulineribeyre) [inline docs/testplans/study-viewer/study-viewer.md:36]: formatting is weird, missing a newline:
Screen Shot 2021-04-28 at 10 29 07 AM

**Review by paulineribeyre — APPROVED** [review]

**Pull request merged**